src/core/time.d

```diff
@@ -4714,34 +4719,29 @@ double _abs(double val) @safe pure nothrow @nogc
   +/
 string numToString(long value) @safe pure nothrow
```

Comment:
I am almost done with it :) I've added signed versions, and also versions that don't require a user-supplied buffer (but do not allocate, a la Walter's dconf15 talk).